README.md

```diff
@@ -25,23 +25,27 @@ if __name__ == "__main__":
 
 ## Installation
 
-The client library is available as a [PyPI package](https://pypi.python.org/pypi/ably).
+### Via PyPI
 
-[Requirements](https://github.com/ably/ably-python#requirements)
+The client library is available as a [PyPI](https://pypi.python.org/pypi/ably) package.
 
-### From PyPI
-
-    pip install ably
+```
+pip install ably
+```
 
 Or, if you need encryption features:
 
-    pip install 'ably[crypto]'
+```
+pip install 'ably[crypto]'
+```
 
-### Locally
+### Via GitHub
```

Comment:
Putting on my "Linux packager hat". the difference between installing via pip and setup.py is that pip will install for your user only (unless ran as root) and is tracked by a package manager, while setup.py will by default attempt to install system wide which will fail without root.

The latter is only really relevant to distro packagers but it's still worth documenting. My gripe is that "Via Github" implies the difference is just where it's from but that's not the case. So I think "Via setup.py" would be better. Or "Globally", "Locally", at a stretch.

I realise this may be a bit pedantic but thought I'd explain the reasoning,

Reply:
When I tested these commands out I saw the following...

I did `pip list` and `ably` was not listed. Then I did `python setup.py install` it installed the package and when I did `pip list` again. I saw that `ably` was listed. It looked like `pip` was managing my `setup.py` install. e.g. I was able to install with `python setup.py install` and uninstall with `pip uninstall ably`

I set up `pyenv` when I installed python, which appears to be managing my global python version. I'm wondering if this is why I'm seeing both installation commands behave the same way (e.g. installing to a global pyenv environment). Just trying to figure out what's going on here and how this is working.

Reply:
The main issue is setup.py would try to install to system directories and needs root. A pyenv would change that.